src/schema.ts

```diff
@@ -1245,6 +1246,99 @@ export const typeDefs = gql`
     isFollowing: Boolean
   }
 
+  type ArenaCategoryV2 {
```

Comment:
Skal vi beholde V2-schemen selv etter at nodebb forsvinner?

Reply:
tja, det kan vi vel gjøre som vi vil med, men ser for meg at det er lett at det bare blir liggende.
Personlig så har jeg ingen sterke meninger.